libs/shared/src/config/trigger-workflow.ts

```diff
@@ -0,0 +1,20 @@
+export enum QueueNameEnum {
```

Comment:
maybe it's better to define these in the application generic?

Reply:
That would generate a circular dependency between `@novu/testing` and `@novu/application-generic`.